coralproject/cay
# save and load a Search

## What does this PR do?

This PR enables saving/loading searches to Pillar. https://github.com/coralproject/cay/issues/192

## How do I test this PR?

- go to the Search Creator screen
- drag some filters
- click Save Search
- enter a name, description and a livetag
- click save
- ensure the new Search shows up on the Saved Searches page

### TODO

- [x] close the modal once query is saved
- [x] redirect to Saved Search Detail (optionally)
- [x] get missing fields from Pillar
- [x] delete a saved search
- [x] show pertinent filters in saved search list

@coralproject/frontend
